assets/src/components/v2/arrow.tsx

```diff
@@ -15,6 +15,7 @@ enum Direction {
   SW = "sw",
   W = "w",
   NW = "nw",
+  U = "u",
```

Comment:
Either name `UTURN =  uturn` or add a comment here so that it's clear this `u` represents a u-turn. I think this would be unclear to someone looking at this without context